sopel/cli/config.py

```diff
@@ -129,7 +136,14 @@ def handle_init(options):
 
 
 def handle_get(options):
-    """Read the settings to display the value of <section> <key>"""
+    """Read the settings to display the value of <section> <key>.
```

Comment:
```suggestion
    """Read the settings to display the value of ``<section> <key>``.
```

Or should it be:

```suggestion
    """Read the settings to display the value of ``<section>`` ``<key>``.
```

It does seem weird not to monospace these.